src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -40,7 +40,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
         EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
 
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedPerson);
+        String expectedMessage = EditCommand.MESSAGE_EDIT_PERSON_SUCCESS + editedPerson;
```

Comment:
I guess what Jessica means is that the output should just be Person edited, and hence you don't need the String.format and editedPerson

Reply:
Line changed to `String expectedMessage = EditCommand.MESSAGE_EDIT_PERSON_SUCCESS;`